docs/getting_started/running_a_workflow_locally.md

```diff
@@ -0,0 +1,94 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+# Running a workflow locally
+
+## Running a workflow in a local Python environment
+
+To quickly test changes to workflow code without the overhead of running a local cluster, you can run a workflow in a local Python environment.
+
+### Prerequisites
+
+* {doc}`Install development tools <installing_development_tools>`
+* {doc}`Create a Flyte project <creating_a_flyte_project>`
+
+### Steps
+
+1. On the command line, navigate to the workflows directory of your Flyte project:
+```{prompt} bash $
+cd my_project/workflows
+```
+2. Run the workflow on the Flyte cluster with `pyflyte run`:
+```{prompt} bash $
+pyflyte run --remote example.py wf
```

Comment:
This assumes `flytectl demo start` has run... I'm not sure this is clear here... or did you mean to just use `pyflyte run example.py wf` ? 
I think emphasizing the fact that the same code runs in a regular python environment to allow iterating super quick and then jumping to `pyflyte run --remote` to run on a local/remote cluster (potentially with more resources) is the story to tell...